### DISCUSSION TITLE
question on grouping MR

### DISCUSSION THREAD

**Discussion by ramarro123** [renovatebot/renovate#26486] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### What is your question?

hi,

first of all, hope you can forgive my english, i know it's not very good :(

i am trying to configure renovate to create a MR that contains all changes of level < major, grouped togheter, and totally skip the major stuff (not even create a MR for them)

so far i did set RENOVATE_PACKAGE_RULES env as 

`
[{"matchUpdateTypes":["major"],"enabled":false},{"matchUpdateTypes":["patch","minor","digest"],"groupName":"non-major dependencies"},{"matchDatasources":["maven"],"registryUrls":["internalgitlabmaven","internalnexusmaven"]}]
`

and it (seems) to work in most of cases, including the usage of my 2 internal registryUrls for maven

what happen is that, if i have some dependencies like log4j2 i got 2 mr.

one called
[[RNVT] Update log4j2 monorepo to v2.22.1]

and the other
[[RNVT] Update non-major dependencies]
(note this mr work as intended, as i can  see dependency from httpclient apache and other from internal libs, so everything look grouped togheter)

the 2nd contains various stuff, grouped correctly. log4j2 seems to get treat as separate group.

can i change this behaviour?

my default.json look like

```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "recreateWhen": "always",
  "extends": [
    ":dependencyDashboard",
    ":semanticPrefixFixDepsChoreOthers",
    ":ignoreModulesAndTests",
    "group:monorepos",
    "group:recommended",
    "replacements:all",
    "workarounds:all"
  ],
  "prConcurrentLimit": 0,
  "prHourlyLimit": 0
}
```

any idea where i should look at? thankyou!

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by secustor** [comment, marked as answer]
Presets take precedence, remove `group:monorepos` from your extend or use [`ignorePresets`](https://docs.renovatebot.com/configuration-options/#ignorepresets)

**Reply by ramarro123** [reply]
worked perfectly thanks!

another note, i don't know if is better to open multiple discussion, but since i am here... :)

i would like to disable this part in the MR
----> disable printing config
Configuration
📅 Schedule: Branch creation - At any time (no schedule defined), Automerge - At any time (no schedule defined).
🚦 Automerge: Disabled by config. Please merge this manually once you are satisfied.
♻ Rebasing: Whenever MR becomes conflicted, or you tick the rebase/retry checkbox.
👻 Immortal: This MR will be recreated if closed unmerged. Get [config help](https://github.com/renovatebot/renovate/discussions) if that's undesired.


---> disable this checkbox
 If you want to rebase/retry this MR, check this box



another question, it's possible to know all the included things in config:recommended so i can exclude what i don't need?

**Reply by viceice** [reply]
please one question per discussion.
